apps/anomaly-detection/README.md

```diff
@@ -10,31 +14,26 @@ The application consists of the following modules:
 * **Model training:** Trains a deep learning model
 * **Model publisher:** Publishes the trained model and some useful statictics to Kafka to be used by the model server
 * **Model serving:** Serving models in real time and updating them as new models will become available. It's based on the
-[minibook](https://www.lightbend.com/blog/serving-machine-learning-models-free-oreilly-ebook-from-lightbend). This module is here for demonstration purposes only. It is absolete and not in use any more.
-* **Speculative model serving:** Serving models, leveraging speculative execution and voting decision making,
-in real time and updating them as new models will become available. Based on this [blog post](https://developer.lightbend.com/blog/2018-05-24-speculative-model-serving/index.html)
-
+[minibook](https://www.lightbend.com/blog/serving-machine-learning-models-free-oreilly-ebook-from-lightbend). This module is here for demonstration purposes only. It is obsolete and not in use any more.
+* **Speculative model serving:** Serving models, leveraging speculative execution and picking results by _voting_. Models are updated as new models will become available. Based on this [blog post](https://developer.lightbend.com/blog/2018-05-24-speculative-model-serving/index.html)
 
 # Installation
 
-Current version of the application is supported only on DC/OS.
-
 ## Installing Data Publisher and Speculative model server
 
 These two modules can be installed either on DC/OS or Kubernetes on DC/OS.
 
 For installation on DC/OS use this [json configuration](/apps/anomaly-detection/source/core/adpublisher/src/main/resources/adpublisher.json)
 for publisher and  [this one](/apps/anomaly-detection/source/core/adspeculativemodelserver/src/main/resources/adspeculativemodelserver.json) for speculative model server.
 
+On OpenShift and Kubernetes, these two modules are installed as 2 pods using the enclosed [Helm Chart](/apps/anomaly-detection/helm).
 
-On kubernetes on DC/OS these two modules are installed as 2 pods on Kubernetes or OpenShift using enclosed [Helm Chart](/apps/anomaly-detection/adchart).
-Make sure you update [values](/apps/anomaly-detection/adchart/values.yaml) to make sure
-it adheres to your environment. Definitions for values can be found [here](/apps/anomaly-detection/adchart/values-metadata.yaml).
+Make sure you update the [values.yaml](/apps/anomaly-detection/helm/values.yaml) to match your environment. The definitions for the values can be found [here](/apps/anomaly-detection/helm/values-metadata.yaml).
 
+## Installing All Training Modules
 
-## Installing all Training Modules
+The current version of the training modules is supported only on DC/OS.
```

Comment:
Thats wrong. It works on OS, see http://grafana-fdp.ibm.lightbend.com/d/cSYlQkXmz/cpu-signal-processing?refresh=30s&orgId=1&from=now-5m&to=now. There is also a chart for installing it on OS

Reply:
Removed paragraph